gateway/tests/sca_actor_test.rs

```diff
@@ -7,9 +7,8 @@ use fvm_shared::clock::ChainEpoch;
 use fvm_shared::econ::TokenAmount;
```

Comment:
Cam we rename this file `gatewa_test.rs` instead of using the outdated `sca` term? I completely missed it till now.